docs/typelibrary/iec61131-3/timer/FB_TON.md

```
@@ -1 +1,54 @@
 # FB_TON
```

markdownlint-cli2 (0.18.1)

Required heading structure
Expected: # Head; Actual: # ADS_SERVER_CONFIG

(MD043, required-headings)